pex/pip/vcs.py

```diff
@@ -50,7 +84,7 @@ def digest_vcs_archive(
     # `pip._internal.req.req_install.InstallRequirement.archive`.
     with TRACER.timed(
         "Digesting {archive} {vcs} archive".format(archive=os.path.basename(archive_path), vcs=vcs)
-    ), temporary_dir(cleanup=False) as chroot, open_zip(archive_path) as archive:
+    ), temporary_dir() as chroot, open_zip(archive_path) as archive:
```

Comment:
Why this change?

Reply:
Unrelated to this change to be sure, but a long standing lack of cleanup from some bug hunt in the past that I spent 15 minutes scrutinizing for, in the end, no reason. No more!